### ISSUE TITLE
High Residuals of Parallel PINN over Vanilla PINN

### ISSUE THREAD

**Issue by raghavg97** [SoumikSarkar830/Laser_PINN#4]
Initial Observation (do not know yet if that’s the reason): 
While calculating residual,
``res_T = u * T_x + v * T_y - alpha * (T_x_x + T_y_y)``, 

**In parallel PINN**, you detach ``u`` and ``v``. Not sure of the reasoning behind that. 
 `` # Get u,v as constants on device_T ``
``u = net_u(X_f_u).detach().to(device_T)``
``v = net_v(X_f_v).detach().to(device_T)``

But **in vanilla PINN**, it is still on the graph (i.e., backpropagated).
